linters/git/.gitlint

```diff
@@ -1,5 +1,5 @@
 [general]
-ignore=body-is-missing
+ignore=body-is-missing, body-max-line-length
```

Comment:
i think we're using 140 line limit in all of our code files? are we sure we want to disable this check?

Reply:
We can leave it.  I was only thinking of removing it since some of the dependabot pr was hitting it.